Parsers/ASimUserManagement/Parsers/ASimUserManagementMicrosoftWindowsEvent.yaml

```diff
@@ -0,0 +1,160 @@
+Parser:
```

Comment:
can you please test filter for this parser from ASC portal?

Reply:
done